src/test/java/ca/corefacility/bioinformatics/irida/pipeline/upload/galaxy/integration/GalaxyToolDataServiceIT.java

```diff
@@ -0,0 +1,285 @@
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tbrucei927_4.0.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tbrucei927_5.0.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tbgambi.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tbgambi_4.0.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tcas_2.0.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/tcas_3.0.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/T_gondii_me49.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/SL2.31.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/SL2.40.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/ITAG2.3.genome",
+                    "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/CCAF000000000.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/vvinifera.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/VcholeraeN16961.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/xenTro2.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/xenTro9.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/laevis_7.1.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/taeGut1.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/zebrafish.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/danRer6.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/danRer7.genome",
+                    "http://s3.amazonaws.com/igv.broadinstitute.org/genomes/danRer10.genome",
+                    "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/AGPv3.31.genome"
+            )
+    );
+
+
+    /**
+     * Sets up for tool data table tests.
+     */
+    @Before
+    public void setup() {
+        galaxyInstanceAdmin = localGalaxy.getGalaxyInstanceAdmin();
+        ToolDataClient toolDataClient = galaxyInstanceAdmin.getToolDataClient();
+        galaxyToolDataService = new GalaxyToolDataService(toolDataClient);
+
+    }
+
+    /**
+     * Tests getting a valid workflow input id from a workflow details.
+     * @throws GalaxyDatasetNotFoundException
+     */
+    @Test
+    public void testGetToolDataTableValid() throws GalaxyDatasetNotFoundException {
+        TabularToolDataTable toolDataTable;
+        try {
+            toolDataTable = galaxyToolDataService.getToolDataTable(VALID_TOOL_DATA_TABLE_ID);
+        } catch (Exception e) {
+            throw new GalaxyDatasetNotFoundException(e);
```

Comment:
Instead of wrapping exceptions in the test code you can just have the method `testGetToolDataTableValid()` throw `Exception` and leave out the `try .. catch` block.

Reply:
Looks good.